demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -667,8 +667,16 @@ export class DemoMeetingApp
       new AsyncScheduler().start(async () => {
         if (this.toggleButton('button-pause-content-share')) {
           this.audioVideo.pauseContentShare();
+          if (this.contentShareType === ContentShareType.VideoFile) {
+            const videoFile = document.getElementById('content-share-video') as HTMLVideoElement;
```

Comment:
Just a thought, should this fix be in the SDK itself and not in the demo app?

Reply:
The video element is in the demo so it is not possible to fix in the SDK. Streaming a video file is just basically giving a media stream to SDK and we already did it in SDK.

Reply:
Gotcha!